frontend/src/routes/Applications/Overview.tsx

```diff
@@ -998,13 +998,13 @@ export default function ApplicationsOverview() {
           },
           {
             id: 'create-argo',
-            text: 'ApplicationSet',
+            text: t('ApplicationSet'),
```

Comment:
I commented on the issue - we may not want to translate this one and `Subscription`.